Dockerfile

```diff
@@ -17,7 +17,7 @@ RUN echo "deb http://nginx.org/packages/debian/ ${NGINX_CODENAME} nginx" >> /etc
 	nginx
 
 # install PM2
-RUN npm i pm2 -g
+RUN yarn global add pm2
```

Comment:
Is yarn dep exist in this container? This is just node container node does not have yarn by default

Reply:
Well yarn comes with node package but I am not sure about the global but if anything went wrong feel free to change.